ehrapy/_utils_available.py

```diff
@@ -4,7 +4,7 @@
 from subprocess import PIPE, Popen
 
 
-def _check_module_importable(package: str) -> bool:  # pragma: no cover
+def check_module_importable(package: str) -> bool:  # pragma: no cover
```

Comment:
I read your justification of why you think that these methods should be public across ehrapy. The reason why I thought that they should be underscore methods is because we don't want users to access these. They are not public API. Now currently they are still in a private module so there'd be some safe guard but I'm not sure whether we need to make them public across ehrapy.

Happy to be convinced otherwise though!